src/torch_admp/base_force.py

```
@@ -110,14 +110,16 @@ def forward(
         # Call the implementation in subclasses
         # Use getattr to explicitly access the tensor and avoid type checker issues
         length_coeff = getattr(self.const_lib, "length_coeff")
```

Ruff (0.14.13)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)